index.html

```diff
@@ -8,49 +9,253 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
+
 <body>
-    
+
     <header>
-        <a href="#" class="logo"> <./ Vi$hal> </a>
-       
+        <a href="#" class="logo">
+            <. / Vi$hal>
+        </a>
+
         <nav class="burger">
             <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
         </nav>
     </header>
-        <section class="content">
-            <div class="introduction">
-                <h1 class="intro">I’m Vishal. <br> Glad to see <br> you!</h1>
-                <p class="description">I’m a software developer! I can help you build a product , feature or website Look
-                    through some of my work and experience! If you like what you see and have a project you need coded,
-                    don’t hestiate tocontact me.</p>
+    <section class="content">
+        <div class="introduction">
+            <h1 class="intro">I’m Vishal. <br> Glad to see <br> you!</h1>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate tocontact me.</p>
+        </div>
+
+        <div class="social-media">
+            <h3 class="contact">LET'S CONNECT </h3>
+            <ul class="social-link">
+                <li class="tweet"><a href="#">
+                        <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-github fa-2x" aria-hidden="true"></i>
+                    </a>
+                <li><a href="#">
+                        <i class="fa fa-angellist fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+
+    </section>
+
+    <section class="Projects">
```

Comment:
As I mentioned in my previous review: 
> Great job trying to implement the request changes 👍 💪
> 
> * [ ]  Please, the project section should use `grid` to lay out elements. This means that `<section class="Projects">` is the one that is supposed to use grid, not the `<div class="card">` (this one should use `flex-box`)
> 
> This will make your project way more maintainable as the grid is an incredible 💪 😮 tool for responsiveness 😄


- [ ] Please use `grid` to layout element **inside** the work section. You have done a great job updating the `card` to use flex-box. but you still need to use `grid` on the section. To summarize, `grid` should be used to layout (positioning) cards, which should use `flex-box` to layout (positioning) elements inside the card. So please update the `<section class="Projects">` to use `grid`. This will make your code look much more responsive 📱 and maintainable 😄